MvvmCross/IoC/MvxTypeExtensions.cs

```diff
@@ -239,5 +239,54 @@ public static object CreateDefault(this Type type)
 
             return Activator.CreateInstance(type);
         }
+        
+        public static ConstructorInfo FindApplicableConstructor(this Type type, IDictionary<string, object> arguments)
+        {
+            var constructors = type.GetConstructors();
+            if (arguments == null || arguments.Count == 0)
+            {
+                return constructors.FirstOrDefault();
```

Comment:
This assumes the parameter-less constructor is always first - is this always true??

Reply:
No, I should change this to null.

Reply:
I was wrong, changed this to the constructor with the least amount of parameters